docs/100_producten/100_borgstelling/borgstelling-beeindigen.feature

```diff
@@ -1,14 +1,27 @@
 # language: nl
 Functionaliteit: Borgstelling beëindigen
 
-  Achtergrond: 
-    Gegeven het kenmerk "093e08db-6791-454b-a172-068099514907"
+  Achtergrond:
+    Gegeven een kredietbank
 
-  Scenario: Borgstelling beëindigen
+  Scenario: Borgstelling beëindigen van afgegeven aanvraag
     Gegeven een afgegeven borgstelling
+    En een beeindigdatum van "2022-01-01"
     Wanneer het "beëindiging borgstelling" bericht is verstuurd via het Schuldenknooppunt
-    En het Schuldenknooppunt bericht is verwerkt
-    Dan is de status van de borgstelling "beëindigd"
+    En het "beëindiging borgstelling" is verwerkt
```

Comment:
```suggestion
    En het Schuldenknooppunt bericht is verwerkt
```

Reply:
Deze regel is specifiek "beeindiging borgstelling" i.v.m. wat process die in een bepaalde status moeten komen. In de toekomst kunnen we dit veranderen, maar voor nu was dit even een snellere optie om dit werkend te krijgen.